SPECS/xapi.spec

```diff
@@ -549,11 +555,7 @@ mkdir $RPM_BUILD_ROOT/etc/xapi.conf.d
 mkdir $RPM_BUILD_ROOT/etc/xcp
 
 mkdir -p %{buildroot}/etc/xenserver/features.d
-%if %{with corosync3}
-echo 1 > %{buildroot}/etc/xenserver/features.d/corosync3
-%else
-echo 0 > %{buildroot}/etc/xenserver/features.d/corosync3
-%endif
+echo 0 > %{buildroot}/etc/xenserver/features.d/hard_numa
```

Comment:
Why is this necessary? It's enabled by default and we need to disable it this way?

By the way, do we support `/etc/xenserver/features.d` configuration at all? And if not, is there something we should do in `xcp-featured` to avoid enabling something that we don't want?

Reply:
> Why is this necessary? It's enabled by default and we need to disable it this way?

This is part of the mechanism to add experimental features that don't need to be added to feature.ml They are disabled by default, but the script that controls these needs a file to enable or disable the feature. The script and the feature daemon in Xenserver work together to create features dynamically. (I assumed this mechanism was available in xcp-ng as well when adding using experimental features) There's code from 2017 in xenopsd referencing the directory, for example.